packages/shared/src/components/profile/Header.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { CSSProperties, ReactElement } from 'react';
+import classNames from 'classnames';
+import { PublicProfile } from '../../lib/user';
+import ShareIcon from '../icons/Share';
+import SettingsIcon from '../icons/Settings';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/ButtonV2';
+import { useShareOrCopyLink } from '../../hooks/useShareOrCopyLink';
+import { ProfilePicture } from '../ProfilePicture';
+import { largeNumberFormat } from '../../lib/numberFormat';
+
+export type HeaderProps = {
+  user: PublicProfile;
+  isSameUser: boolean;
+  sticky?: boolean;
+  className?: string;
+  style?: CSSProperties;
+};
+
+export function Header({
+  user,
+  isSameUser,
+  sticky,
+  className,
+  style,
+}: HeaderProps): ReactElement {
+  const [, onShareOrCopyLink] = useShareOrCopyLink({
+    text: `Check out ${user.name}'s profile on daily.dev`,
+    link: user.permalink,
+    trackObject: () => ({ event_name: 'share profile', target_id: user.id }),
+  });
+
+  return (
+    <header
+      className={classNames('flex items-center px-4 h-12', className)}
+      style={style}
```

Comment:
We can't do this with tailwind?

Reply:
I see that you already figured it out. So I guess it's ok :)